src/components/admin/order/OrderStickyNavBar.tsx

```diff
@@ -51,10 +51,26 @@ const ShareButton = styled(ShareSvg)`
 interface OrderStickyNavBarProps {
   showNavBar: boolean;
   workspaceName: string;
-  onClickShare: () => void;
 }
 
-function OrderStickyNavBar({ showNavBar, workspaceName, onClickShare }: OrderStickyNavBarProps) {
+function OrderStickyNavBar({ showNavBar, workspaceName }: OrderStickyNavBarProps) {
+  const onClickShare = async () => {
+    if (!navigator.share) {
+      alert('이 브라우저는 Web Share API를 지원하지 않습니다. 다른 브라우저를 사용해주세요.');
+      return;
+    }
+
+    try {
+      await navigator.share({
+        title: document.title,
```

Comment:
```suggestion
        title: workspaceName,
```

이거 document.title 넘기면 ''키오스쿨'' 밖에 안뜨지 않나요??

Reply:
지난번에 title을 각 페이지에 맞춰서 바꾸는 작업 진행하는 걸로 알고 있어서 해당 사항을 대비하여 title로 적어 놓았습니다!

Reply:
yagni(you ain't gonna need it) 미래에 반영될 부분을 생각해서 미리 작업해두는 것을 하지말라라는 뜻인데요.

이런 방식이 나중에 기획이 바뀌거나 했을 때 방해가 되고, 또한 현재 구현의 목적과 목표가 희미해지게 되는 결과를 만들 수도 있어서, 미래를 생각하여 미리 반영하는 방식은 지양해주시면 좋을 것 같네요~

Reply:
넵~